packages/pull-request-automation/src/add-first-time-contributor-label.js

```diff
@@ -0,0 +1,20 @@
+export default async function addFirstTimeContributorLabel( context, octokit ) {
+	const owner = context.payload.repository.owner.login;
+	const repo = context.payload.repository.name;
+	const author = context.payload.pull_request.user.login;
+
+	const { total_count: totalCount } = await octokit.search.commits( {
+		q: `repo:${ owner }/${ repo }+author:${ author }`,
+	} );
+
+	if ( totalCount !== 0 ) {
+		return;
```

Comment:
Would be cool to have some extra logging output around early returns, as it really helps work out why an action did something/nothing.

Reply:
Added a tonne of logging in c22a98e0ea1419abcfee2072b9617ef509185064 👌